components/match2/commons/player_ext.lua

```diff
@@ -105,7 +105,6 @@ function PlayerExt.fetchTeamHistoryEntry(resolvedPageName, date)
 		'([[extradata_joindate::<' .. date .. ']] or [[extradata_joindate::' .. date .. ']])',
 		'[[extradata_joindate::>]]',
 		'[[extradata_leavedate::>' .. date .. ']]',
-		'[[extradata_position::player]]',
```

Comment:
And positive that this will not have any negative impact?

Reply:
![Screenshot 2023-01-27 12 37 06](https://user-images.githubusercontent.com/75081997/215077314-cba1db8d-044e-4fcf-be3e-02aa69b24379.png)